docker-compose/cassandra-3.11/.env

```diff
@@ -0,0 +1,3 @@
+COMPOSE_PROJECT_NAME=cass311-stargate
+CASSTAG=3.11.11
```

Comment:
Can you extend the `README.md` in the `persistence-cassandra-3.11` and add the update of this value to the list of tasks we need to do when upgrading the version?

Reply:
fixing